pytorch_forecasting/models/base_model.py

```diff
@@ -911,13 +911,19 @@ def configure_optimizers(self):
                     self.parameters(), lr=lr, weight_decay=self.hparams.weight_decay, **optimizer_params
                 )
             except TypeError:  # in case there is no weight decay
-                optimizer = self.optimizer(self.parameters(), lr=lr, **optimizer_params)
+                optimizer = self.optimizer(
+                    self.parameters(), lr=lr, weight_decay=self.hparams.weight_decay, **optimizer_params
```

Comment:
please remove the weight_decay parameter here as it might not exist for all optimizers

Reply:
Done!